src/dss/manifest_templates/mlflow_deployment.yaml.j2

```diff
@@ -1,19 +1,12 @@
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: dss
-  labels:
-    app.kubernetes.io/part-of: dss
 
 ---
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: mlflow
-  namespace: dss
+  name: {{ name }}
```

Comment:
If in the future we add some other manifest files that need to be render with a `name` and `namespace` different than these, things will collide. I think it would be nice to be clear about this context variables, maybe prefix them with `mlflow_`?

Reply:
yeah this is a good point.  If they're rendered by different KRH's then it doesn't really matter, but part of the KRH conveneince is bundling them all together.  I can add prefixes